src/data/services/lms/hooks/api.js

```diff
@@ -0,0 +1,47 @@
+import { useQuery } from 'react-query';
+
+import { camelCaseObject } from '@edx/frontend-platform';
+
+import { queryKeys } from '../constants';
+import { loadORAConfigData, loadSubmissionData } from '../dataLoaders';
+import fakeData from '../fakeData';
+
+/**
+ *  A react-query data object
+ *  @typedef {Object} ReactQueryData
```

Comment:
[curious] I wonder if `@tanstack/react-query` exports any existing types for `useQuery`?

Reply:
was curious about that as well, but did not see something immediately.  Not sure the best way to import jsdoc types...

Reply:
If `@tanstack/react-query` exports its types, you should be able to import it like any other module:

```js
import { UseQueryResult } from '@tanstack/react-query';
```
It looks like there is are types defined for its hooks: https://github.com/TkDodo/react-query/blob/c1ae82ba188fd5abda5e256cac070145e5941447/src/react/useQuery.ts#L16

The types for the hooks do appear to be exported: https://github.com/TkDodo/react-query/blob/c1ae82ba188fd5abda5e256cac070145e5941447/src/react/index.ts#L19

Only question if if JSDoc would accept a TS-defined type 😄